examples/compose/plot_column_transformer_mixed_types.py

```diff
@@ -71,9 +71,6 @@
 clf = Pipeline(steps=[('preprocessor', preprocessor),
                       ('classifier', LogisticRegression())])
 
-X = data.drop('survived', axis=1)
```

Comment:
I would like to document this idiom somewhere. It's sooo common and I'm not sure if it's anywhere else in the docs.
Maybe give it as an alternative here?

Reply:
Added a comment using the `frame` attribute.

Reply:
thanks!

Reply:
> I would like to document this idiom somewhere.

Is `y = X.pop('survived')` a better idiom? Or not because it modifies X in-place.

Reply:
I personally prefer `frame.drop(columns='survived')` instead of specifying the ``axis=1``